lucene/core/src/java/org/apache/lucene/search/PointRangeQuery.java

```diff
@@ -66,29 +63,7 @@ public abstract class PointRangeQuery extends Query {
    *     upperValue.length}
    */
   protected PointRangeQuery(String field, byte[] lowerPoint, byte[] upperPoint, int numDims) {
-    checkArgs(field, lowerPoint, upperPoint);
-    this.field = field;
-    if (numDims <= 0) {
-      throw new IllegalArgumentException("numDims must be positive, got " + numDims);
-    }
-    if (lowerPoint.length == 0) {
-      throw new IllegalArgumentException("lowerPoint has length of zero");
-    }
-    if (lowerPoint.length % numDims != 0) {
-      throw new IllegalArgumentException("lowerPoint is not a fixed multiple of numDims");
-    }
-    if (lowerPoint.length != upperPoint.length) {
-      throw new IllegalArgumentException(
-          "lowerPoint has length="
-              + lowerPoint.length
-              + " but upperPoint has different length="
-              + upperPoint.length);
-    }
-    this.numDims = numDims;
-    this.bytesPerDim = lowerPoint.length / numDims;
-
-    this.lowerPoint = lowerPoint;
-    this.upperPoint = upperPoint;
+    config = new PRQConfig(field, lowerPoint, upperPoint, numDims);
```

Comment:
We probably want to keep it as `PointRangeQueryConfig` just to be inline with the rest of the naming